src/main/java/plugily/projects/villagedefense/arena/managers/ZombieSpawnManager.java

```diff
@@ -39,6 +39,26 @@
  */
 public class ZombieSpawnManager {
 
+  private static final List<CustomZombieSpawnManager> CUSTOM_ZOMBIE_SPAWN_MANAGERS = new ArrayList<>();
+
+  /**
+   * Add a custom zombie spawn manager
+   *
+   * @param customZombieSpawnManager the manager
+   */
+  public static void addCustomZombieManager(CustomZombieSpawnManager customZombieSpawnManager) {
+    CUSTOM_ZOMBIE_SPAWN_MANAGERS.add(customZombieSpawnManager);
+  }
+
+  /**
+   * Remove a custom zombie spawn manager
+   *
+   * @param customZombieSpawnManager the manager
+   */
+  public static void removeCustomZombieManager(CustomZombieSpawnManager customZombieSpawnManager) {
+    CUSTOM_ZOMBIE_SPAWN_MANAGERS.remove(customZombieSpawnManager);
+  }
```

Comment:
You could remove these two methods and make the `CUSTOM_ZOMBIE_SPAWN_MANAGERS` property `public`.